packages/postcss/__tests__/integration.test.js

```diff
@@ -0,0 +1,93 @@
+import test from 'ava'
+import { css } from '@webpack-blocks/assets'
+import { createConfig, match } from '@webpack-blocks/core'
+import postcss from '../index'
+
+function LoaderOptionsPlugin (loaderOptions) {
+  this.loaderOptions = loaderOptions
+}
+
+test('Postcss works with defaults, without match()', t => {
+  const config = createConfig({}, [
+    postcss()
+  ])
+
+  t.deepEqual(config.plugins, [])
+  t.deepEqual(config.module.rules, [
+    {
+      test: /\.css$/,
+      use: [
+        'style-loader',
+        'css-loader',
+        {
+          loader: 'postcss-loader',
+          options: {}
+        }
+      ]
+    }
+  ])
+})
+
+test('Postcss works with css() & match()', t => {
```

Comment:
Done. But I added it to the global integration test (`packages/webpack/__tests__/integration.test.js`).